admin_manual/configuration/occ_command.rst

```diff
@@ -0,0 +1,252 @@
+* Query and change LDAP settings
+
+``occ`` is in the :file:`owncloud/` directory; for example 
+:file:`/var/www/owncloud` on Ubuntu Linux. ``occ`` is a PHP script. The 
+preferred way to run it is as your HTTP user. Running it with no options lists 
+all commands and options, like this example on Ubuntu:: 
+
+ $ sudo -u www-data php occ
+
+This is the same as ``sudo -u www-data php occ list``.
+ 
+Run it with the ``-h`` option for syntax help::
+
+ $ sudo -u www-data php occ -h
+ 
+Display your ownCloud version::
+
+ $ sudo -u www-data php occ -V
+   ownCloud version 7.0.4
+   
+Query your ownCloud server status::
+ 
+ $ sudo -u www-data php occ status
+   Array
+   (
+    [installed] => true
+    [version] => 7.0.4.2
+    [versionstring] => 7.0.4
+    [edition] => 
+   )
+   
+``occ`` has options, commands, and arguments. Options and arguments are 
+optional, while commands are required. The syntax is::
+
+ occ [options] command [arguments]
+ 
+Get detailed information on individual commands with the ``help`` command, like 
+this example for the ``maintenance:mode`` command::
+ 
+ $ sudo -u www-data php occ help maintenance:mode
+   Usage:
+   maintenance:mode [--on] [--off]
+
+   Options:
+   --on                  enable maintenance mode
+   --off                 disable maintenance mode
+   --help (-h)           Display this help message.
+   --quiet (-q)          Do not output any message.
+   --verbose (-v|vv|vvv) Increase the verbosity of messages: 1 for normal 
+   output, 2 for more verbose output and 3 for debug
+   --version (-V)        Display this application version.
+   --ansi                Force ANSI output.
+   --no-ansi             Disable ANSI output.
+   --no-interaction (-n) Do not ask any interactive question.
+   
+Maintenance Commands
+--------------------
+
+These three maintenance commands put your ownCloud server into three modes: maintenance, 
+singleuser, and repair.
+
+You must put your ownCloud server into maintenance mode whenever you perform an 
+update or upgrade. This locks the sessions of all logged-in users, including 
+administrators, and displays a status screen warning that the server is in 
+maintenance mode. Users who are not already logged in cannot log in until 
+maintenance mode is turned off. When you take the server out of maintenance mode 
+logged-in users must refresh their Web browsers to continue working::
+
+ $ sudo -u www-data php occ maintenance:mode --on
+ $ sudo -u www-data php occ maintenance:mode --off
+ 
+Putting your ownCloud server into single-user mode allows admins to log in and 
+work, but not ordinary users. This is useful for performing maintenance and 
+troubleshooting on a running server::
+
+ $ sudo -u www-data php occ maintenance:singleuser --on
+   Single user mode enabled
+   
+And turn it off when you're finished::
+
+ $ sudo -u www-data php occ maintenance:singleuser --off
+   Single user mode disabled
+
+TODO: What does  maintenance:repair do? Needs details::
```

Comment:
It's a set of repair task - this will change over time. Generally speaking these task help to clean up the database. We run them automatically during upgrade and it is possible to manually execute them - if needed.